Firebase/CoreDiagnostics/FIRCDLibrary/FIRCoreDiagnostics.m

```diff
@@ -153,7 +152,7 @@ @interface FIRCoreDiagnostics : NSObject <FIRCoreDiagnosticsInterop>
 @property(nonatomic, readonly) GDTCORTransport *transport;
 
 /** The storage to store the date of the last sent heartbeat. */
-@property(nonatomic, readonly) FIRCoreDiagnosticsDateFileStorage *heartbeatDateStorage;
+@property(nonatomic, readonly) GULStorageHeartbeat *heartbeatDateStorage;
```

Comment:
The variable name looks like a perfect name for the class itself - `GULHeartbeatDateStorage`, WDYT?